tests/unit/test_bytestring_types.py

```diff
@@ -5,12 +5,12 @@
 from nucypher.characters.lawful import Enrico
 
 
-def test_message_kit_serialization_via_enrico(federated_alice):
+def test_message_kit_serialization_via_enrico(alice):
 
     mock_label = b'this is a label'
 
     # Enrico
-    enrico = Enrico.from_alice(federated_alice, label=mock_label)
+    enrico = Enrico.from_alice(alice, label=mock_label)
```

Comment:
We can modify this unit test to not take the `alice` fixture and therefore not need testerchain/any blockchain ursula creation.

```suggestion
    enrico = Enrico(policy_encrypting_key=SecretKey.random().public_key())
```

Then the `alice` fixture is not needed.